docs/src/modules/java/pages/quickstart/cr-value-entity-scala.adoc

```diff
@@ -0,0 +1,228 @@
+* You'll also need to install the https://developer.lightbend.com/docs/akka-serverless/akkasls/install-akkasls.html[Akka Serverless CLI, window="new-doc"] if you want to deploy from a terminal window.
+* For this quickstart, you'll also need
+** https://docs.docker.com/engine/install[Docker {minimum_docker_version} or higher, window="new"]
+** Java {java-version} or higher
+** https://www.scala-sbt.org/download.html[sbt 1.4 or higher, window="new"]
+** https://github.com/fullstorydev/grpcurl#installation[`grpcurl`, window="new"]
+
+[NOTE]
+====
+If you want to bypass writing code and jump straight to the deployment:
+
+. Download the source code using the Akka Serverless CLI: 
+`akkasls quickstart download customer-registry-scala`
+
+. Skip to <<Package and deploy your service>>.
+====
+
+== Writing the Customer Registry
+
+. From the command line, create a directory for your project.
++
+[source,command line]
+----
+mkdir customerregistry
+----
+
+. Change into the project directory.
++
+[source,command line]
+----
+cd customerregistry
+----
+
+. Download the `build.sbt` file
++
+[source,command line]
+----
+curl -OL https://raw.githubusercontent.com/lightbend/akkaserverless-java-sdk/main/samples/scala-customer-registry-quickstart/build.sbt
+----
+
+. Create the sbt `project` directory
++
+[source,command line]
+----
+mkdir project
+----
+
+. Download the `plugins.sbt`
+[source,command line]
+----
+curl -L https://raw.githubusercontent.com/lightbend/akkaserverless-java-sdk/main/samples/scala-customer-registry-quickstart/project/plugins.sbt -o project/plugins.sbt
+----
+
+
+== Define the external API
+
+The Customer Registry service will create or retrieve a customer, including their email, phone number and mailing address. The `customer_api.proto` will contain the external API your clients will invoke.
+
+. In your project, create two directories for you protobuf files, `src/main/proto/customer/domain` and `src/main/proto/customer/api`.
+[.tabset]
+Linux or macOS::
++
+--
+[source,command line]
+----
+mkdir -p ./src/main/proto/customer/api
+mkdir -p ./src/main/proto/customer/domain
+----
+--
+Windows 10+::
++
+--
+[source,command line]
+----
+mkdir src/main/proto/customer/api
+mkdir src/main/proto/customer/domain
+----
+--
+
+. Create a `customer_api.proto` file and save it in the `src/main/proto/customer/api` directory.
+
+. Add declarations for:
++
+* The protobuf syntax version, `proto3`.
+* The package name, `customer.api`.
+* Import `google/protobuf/empty.proto`.
++
+[source,proto,indent=0]
+.src/main/proto/customer/api/customer_api.proto
+----
+include::example$scala-customer-registry-quickstart/src/main/proto/customer/api/customer_api.proto[tag=declarations]
+----
+
+. Add the service endpoint. The service endpoint is annotated with `akkaserverless.codegen` indicating we want to generate a Value Entity for this service.
```

Comment:
Old annotation name, or should that stay?

Reply:
What do you mean? This is about the new annotation. Code snippet is also showing the new.

Reply:
hmm, I just spotted a left over but in another file. Maybe you commented on the wrong line?